packages/loot-core/src/server/accounts/transaction-rules.ts

```diff
@@ -849,6 +857,13 @@ export async function prepareTransactionForRules(
     }
   }
 
+  if (trans.account) {
+    const account = await getAccount(trans.account);
```

Comment:
Should we have any performance concerns with this approach or is there some internal caching pulling the weight for us?

IMO it would be better to fetch all accounts into an object/map and then just retrieve the right object using the accountId in the transaction instead of a db query for each transaction, what do you think?

Reply:
done @matt-fidd